src/ImageBackendFactory.php

```diff
@@ -37,9 +37,15 @@ public function __construct(Factory $creator)
      */
     public function create($service, array $params = [])
     {
-        /** @var AssetContainer $assetContainer */
+        /** @var AssetContainer|null $assetContainer */
         $assetContainer = reset($params);
-        if (!$assetContainer instanceof AssetContainer) {
+
+        // If no asset container was passed in, create a new uncached image backend
+        if (!$assetContainer) {
+            return $this->creator->create($service, $params);
+        }
```

Comment:
Needed so that developers can get an instance of the injected `Image_Backend` when they don't have a valid `AssetContainer` instance that contains an image file. The example in the "manual testing steps" section of the PR description simply won't work without this.